Fix build warnings CS0108 and CS0618

nopCommerce version: `develop`

Steps to reproduce the problem: 
When you compile the application the following warnings are presented:

```text
\src\Libraries\Nop.Services\Connected Services\EuropaCheckVatService\Reference.cs(328,52): warning CS0108: 'checkVatPortTypeClient.CloseAsync()' hides inherited member 'ClientBase<checkVatPortType>.CloseAsync()'. Use the new keyword if hiding was intended. [\src\Libraries\Nop.Services\Nop.Services.csproj]
\src\Libraries\Nop.Services\Caching\RedisSynchronizedMemoryCache.cs(68,30): warning CS0618: 'RedisChannel.implicit operator RedisChannel(string)' is obsolete: 'It is preferable to explicitly specify a PatternMode, or use the Literal/Pattern methods' [\src\Libraries\Nop.Services\Nop.Services.csproj]
\src\Libraries\Nop.Services\Caching\RedisSynchronizedMemoryCache.cs(131,13): warning CS0618: 'RedisChannel.implicit operatorRedisChannel(string)' is obsolete: 'It is preferable to explicitly specify a PatternMode, or use the Literal/Pattern methods' [\src\Libraries\Nop.Services\Nop.Services.csproj]
\src\Libraries\Nop.Services\Caching\RedisSynchronizedMemoryCache.cs(201,36): warning CS0618: 'RedisChannel.implicit operatorRedisChannel(string)' is obsolete: 'It is preferable to explicitly specify a PatternMode, or use the Literal/Pattern methods' [\src\Libraries\Nop.Services\Nop.Services.csproj]
```

It would be useful to clean these warnings.

--- a/src/Libraries/Nop.Services/Caching/RedisSynchronizedMemoryCache.cs
+++ b/src/Libraries/Nop.Services/Caching/RedisSynchronizedMemoryCache.cs
@@ -65,7 +65,7 @@ public partial class RedisSynchronizedMemoryCache : ISynchronizedMemoryCache
     {
         var channel = ChannelPrefix;
         var subscriber = _connection.GetSubscriber();
-        subscriber.Subscribe(channel + "*", (redisChannel, value) =>
+        subscriber.Subscribe(RedisChannel.Pattern(channel + "*"), (redisChannel, value) =>
         {
             var publisher = ((string)redisChannel).Replace(channel, "");
             if (publisher == _processId)
@@ -127,8 +127,8 @@ public partial class RedisSynchronizedMemoryCache : ISynchronizedMemoryCache
             return;
 
         var subscriber = _connection.GetSubscriber();
-        subscriber.Publish(
-            $"{ChannelPrefix}{_processId}",
+        subscriber.Publish(RedisChannel.Pattern( 
+            $"{ChannelPrefix}{_processId}"),
             JsonConvert.SerializeObject(keys),
             CommandFlags.FireAndForget);
     }
@@ -198,7 +198,7 @@ public partial class RedisSynchronizedMemoryCache : ISynchronizedMemoryCache
         {
             _timer?.Dispose();
             var subscriber = _connection.GetSubscriber();
-            subscriber.Unsubscribe(ChannelPrefix + "*");
+            subscriber.Unsubscribe(RedisChannel.Pattern(ChannelPrefix + "*"));
 
             _disposed = true;
         }

